controllers/virtualmachine/v1alpha2/virtualmachine_controller_suite_test.go

```diff
@@ -10,22 +10,60 @@ import (
 
 	ctrlmgr "sigs.k8s.io/controller-runtime/pkg/manager"
 
+	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha1"
+	vmopv1a2 "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
 	virtualmachine "github.com/vmware-tanzu/vm-operator/controllers/virtualmachine/v1alpha2"
 	ctrlContext "github.com/vmware-tanzu/vm-operator/pkg/context"
 	"github.com/vmware-tanzu/vm-operator/pkg/lib"
+	pkgmgr "github.com/vmware-tanzu/vm-operator/pkg/manager"
 	providerfake "github.com/vmware-tanzu/vm-operator/pkg/vmprovider/fake"
 	"github.com/vmware-tanzu/vm-operator/test/builder"
+	mutationv1a1 "github.com/vmware-tanzu/vm-operator/webhooks/virtualmachine/v1alpha1/mutation"
+	validationv1a1 "github.com/vmware-tanzu/vm-operator/webhooks/virtualmachine/v1alpha1/validation"
+	mutationv1a2 "github.com/vmware-tanzu/vm-operator/webhooks/virtualmachine/v1alpha2/mutation"
+	validationv1a2 "github.com/vmware-tanzu/vm-operator/webhooks/virtualmachine/v1alpha2/validation"
 )
 
 var intgFakeVMProvider = providerfake.NewVMProviderA2()
 
-var suite = builder.NewTestSuiteForControllerWithFSS(
-	virtualmachine.AddToManager,
-	func(ctx *ctrlContext.ControllerManagerContext, _ ctrlmgr.Manager) error {
-		ctx.VMProviderA2 = intgFakeVMProvider
-		return nil
-	},
-	map[string]bool{lib.VMServiceV1Alpha2FSS: true})
+var suite = builder.NewTestSuiteWithOptions(
+	builder.TestSuiteOptions{
+		InitProviderFn: func(ctx *ctrlContext.ControllerManagerContext, _ ctrlmgr.Manager) error {
+			ctx.VMProviderA2 = intgFakeVMProvider
+			return nil
+		},
+		FeatureStates: map[string]bool{lib.VMServiceV1Alpha2FSS: true},
+		Controllers:   []pkgmgr.AddToManagerFunc{virtualmachine.AddToManager},
+		ConversionWebhooks: []builder.TestSuiteConversionWebhookOptions{
+			{
+				Name: "virtualmachines.vmoperator.vmware.com",
+				AddToManagerFn: []func(ctrlmgr.Manager) error{
+					(&vmopv1.VirtualMachine{}).SetupWebhookWithManager,
+					(&vmopv1a2.VirtualMachine{}).SetupWebhookWithManager,
+				},
+			},
+		},
+		MutationWebhooks: []builder.TestSuiteMutationWebhookOptions{
+			{
+				Name:           "default.mutating.virtualmachine.v1alpha1.vmoperator.vmware.com",
```

Comment:
If we're trying to mimic the actual env, the v1a1 webhooks aren't enabled in v1a2.